src/db/tables/subscribers.js

```diff
@@ -277,35 +277,10 @@ async function setAllEmailsToPrimary (subscriber, allEmailsToPrimary) {
 /* c8 ignore stop */
 
 /**
- * OBSOLETE, preserved for backwards compatibility
```

Comment:
Thanks! I'll have to make a ticket to start cleaning up some of these obsolete functions as well as columns in our `subscriber` row. 

Some columns that are no longer being used include:
```
fx_newsletter
breaches_last_shown
breaches_resolved
waitlists_joined
breach_stats
...
```

and all the corresponding util funcs

Reply:
Added: https://mozilla-hub.atlassian.net/browse/MNTOR-2972 

FYI